doc/compatibility_matrix.rst

```diff
@@ -15,27 +15,41 @@ The following table demonstrates the compatibility between the nRF Connect SDK a
      - Amazon specification (Sidewalk MCU SDK version)
      - Supported platforms
      - Amazon qualification status
-   * - v1.14.3
-     - v2.3.0
+   * - ---

```

Comment:
@ktaborowski @RobertGalatNordic could you check the compatibility table? And if there's anything that needs to be linked (the file that you've mentioned), could you add the suggestion as well?

Reply:
no, as far as I'm aware qualification of 2.7.0 is still ongoing so everything is in order.

Reply:
files to be hosted on zoomin, tba